_format/1.1/index.md

```diff
@@ -787,18 +796,12 @@ document for that collection:
 
 ##### <a href="#document-links-link-relation-type" id="document-links-link-relation-type" class="headerlink"></a> Link relation type
 
-The value of the `rel` member **MUST** be a string or an array of strings that
-represent link relation type(s). Valid link relation types are defined by
+The value of the `rel` member **MUST** be a string that represents a link relation
+type. Valid link relation types are defined by
 [RFC8288 Section 2.1](https://tools.ietf.org/html/rfc8288#section-2.1).
 
-If a link is represented as a string, or the `rel` member is not given on a
-link object, the link's relation type **SHOULD** be inferred from the name of
-the link object.
-
-An array of link relationship types establishes multiple links that share the
-same context, target, and target attributes and a client **MUST** treat these
-links as separate, distinct links. A client **MUST NOT** infer additional
-semantics for the link based on a composition of link relation types alone.
+If a `rel` member is not given on a link object, the link's relation type
```

Comment:
Isn't the condition "If a link is represented as a string," (removed above) still appropriate here as well?

Reply:
I don't think so because it **MUST** be a string. I do think this could be rewritten more clearly though. What about this?

"If a link object does not have a `rel` member, the link's relation type **SHOULD** be inferred from the name of the link object."

Reply:
> If a link is represented as a string, or the `rel` member is not given on a
> link object, the link's relation type **SHOULD** be inferred from the name of
> the link object.

My interpretation of the above paragraph is that the clause "If a link is represented as a string" covers links that are not represented by link objects at all, but rather as strings. Such as `"self": "http://example.com/api/posts"`.

Reply:
ahhh, that makes sense. 🙏  thanks for the explanation.